### PR TITLE
Capx 24 [incomplete]

### PR DESCRIPTION
**Pull request by mpriest**
This is at a place to review...

As per title, this is incomplete. Jira has some more details.
Major points of contention in this PR:
- Unnecessary API changes.
- jqTree Apache license requires the CAPx module to be GPLv3.

### CODE REVIEW

File: stanford_capx.docs.inc [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+ * A place to put your documentation related functionality.
+ */
+
+use CAPx\Drupal\Util\CAPxConnection;
+use \Peekmo\JsonPath\JsonStore as JsonParser;
+
+define('STANFORD_CAPX_SAMPLE_PROFILE', 'sample_profile.json');
+
+/**
+ * [stanford_capx_admin_config description]
```
Comment (sherakama) [inline stanford_capx.docs.inc]: Please complete the function documentation.

File: stanford_capx.module [hunk 2]
```diff
@@ -121,6 +124,17 @@ function stanford_capx_menu() {
     'weight' => 10,
   );
 
+  // Data Browser
+  $items['admin/config/capx/data-browser'] = array(
+    'title' => 'Data Browser',
+    'page callback' => 'stanford_capx_admin_config_data_browser',
+    'file' => 'stanford_capx.docs.inc',
+    'access arguments' => array('administer capx'),
+    'type' => MENU_NORMAL_ITEM,
```
Comment (sherakama) [inline stanford_capx.module]: This doesn't need to be in the menu. Please change to MENU_SUGGESTED_ITEM so that site admins can decide.

**Comment by sherakama** [comment]
Looks great. I left a couple of comments inline and one in Jira. Just a couple of small items and this one is done. Nice work.

**Comment by mpriest** [comment]
@sherakama I've added
- schema refresh button
- selectors on objects
- function documentation
- suggest menu item

**Comment by sherakama** [comment]
Rebased and committed: https://github.com/SU-SWS/stanford_capx/commit/7129ac083ebf4409b391cdeee494ca4c2da516a8

Thanks for all of this

**Pull request closed without merging**